api/app/signals/apps/email_integrations/actions/signal_created.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 - 2022 Gemeente Amsterdam
+import logging
+
+from signals.apps.email_integrations.actions.abstract import AbstractAction
+from signals.apps.email_integrations.models import EmailTemplate
+from signals.apps.email_integrations.rules import SignalCreatedRule
+
+logger = logging.getLogger(__name__)
+
+
+class SignalCreatedAction(AbstractAction):
+    rule = SignalCreatedRule()
+
+    key = EmailTemplate.SIGNAL_CREATED
+    subject = 'Bedankt voor uw melding {signal_id}'
+
+    note = 'Automatische e-mail bij registratie van de melding is verzonden aan de melder.'
+
+    def get_additional_context(self, signal):
+        return {
+            'afhandelings_text': signal.category_assignment.category.handling_message
```

Comment:
Now that we have this code assuming the presence of certain things in the email template, we should probably give an example for each action. (If not an outright defaults that are specific to each action.)

Reply:
This code does not assume anything about email templates. It provides additional context to be used in certain email templates. I agree on making this more visible for the end user however this is not the place to do that. I also have a ticket in Amsterdam specifically for this, so I would like to pick this up in a separate PR. In that PR I also want to remove some deprecated email variables and add some new ones if needed.